src/clj/ataru/person_service/person_client.clj

```diff
@@ -23,7 +21,10 @@
                  (resolve-url :oppijanumerorekisteri-service.person-create) person)]
     (match result
       {:status 201 :body body}
```

Comment:
Use also :exists :status when the person was already created. There is (at least in master) code which logs based on :status :exists